modules/dmrpp_module/reduce_mdf.cc

```diff
@@ -100,8 +100,9 @@ short update_sha256_file(char* m_dmrpp_fname,char* m_h5_fname,char* m_sha256_fna
 
     // If the recording file that stores thesha256 doesn't exist, 
     // just create this file and write the sha256 etc information to the file.
-    if(access(m_sha256_fname,F_OK)==-1) 
-        return write_sha256_file(m_dmrpp_fname,m_h5_fname,m_sha256_fname,sha256_buf);
+
+    //if(access(m_sha256_fname,F_OK)==-1)
```

Comment:
Lets clean up these comments. Explain why the above was removed (or just eliminate it), fix the typos in the 'If the recording...' comments and then explain briefly write_sha256_file() should be called if open() fails (because two years from now we'll look at this and go, "Whoa, what's going on here?").

Thanks.

Reply:
added comments